stream-android-core/src/main/java/io/getstream/android/core/api/model/value/StreamHttpClientInfoHeader.kt

```diff
@@ -24,10 +24,10 @@ import kotlin.text.iterator
  *
  * @property rawValue The raw value of the header.
  */
-@StreamCoreApi
+@StreamPublishedApi
 @JvmInline
-value class StreamHttpClientInfoHeader private constructor(val rawValue: String) {
-    companion object {
+public value class StreamHttpClientInfoHeader private constructor(public val rawValue: String) {
```

Comment:
Is this supposed to be used by clients? In feeds it's just used internally to create the core client